apatite/apatite/dal.py

```diff
@@ -317,6 +317,14 @@ def plist_sort_key(project):
         for project in sorted(self.project_list, key=plist_sort_key):
             if not project.desc[-1:] in ').!':
                 project = attr.evolve(project, desc=project.desc + '.')
+
+            cleaned_urls = []
+            for url_name, url in project.urls:
+                # strip off trailing slashes from all urls
+                new_path = tuple([segm for segm in url.path if segm != ''])
+                cleaned_urls.append((url_name, url.replace(path=new_path)))
```

Comment:
you can also normalize the url: https://hyperlink.readthedocs.io/en/latest/api.html#hyperlink.URL.normalize

Reply:
Sounds good :+1: